integration/indexes_compat_test.go

```diff
@@ -316,6 +316,13 @@ func TestIndexesCreateRunCommand(t *testing.T) {
 			indexName:      "custom-name",
 			resultType:     emptyResult,
 		},
+		"empty-collection-name": {
```

Comment:
Let's have our new test cases the way we want to name `EmptyCollectionName` :hugs: